data/rbac_admins_seed.yml

```diff
@@ -0,0 +1,22 @@
+---
+access1: &RBACAdminAccess
```

Comment:
So the `seed_file` argument on the rake task can be either this `admins` version or the `general` catalog seed version?

Reply:
Never mind this question. This looks good to me. I want to try it on the pod.